frontend/app/services/EventbriteService.scala

```diff
@@ -77,6 +79,7 @@ trait EventbriteService extends WebServiceHelper[EBObject, EBError] {
   def getEventsByIds(ids: Seq[String]): Seq[RichEvent] = events.filter(e => ids.contains(e.event.id))
   def getLimitedAvailability: Seq[RichEvent] = events.filter(_.event.isLimitedAvailability)
   def getRecentlyCreated(start: DateTime): Seq[RichEvent] = events.filter(_.created.isAfter(start))
+  def getSortedByCreationDate: Seq[RichEvent] = events.sortBy(_.created.toDateTime).reverse
```

Comment:
This is suboptimal as it involves two traversals of the list. Alternatively, you can supply a reverse ordering and do it in one traversal only
```scala
events.sorted(Ordering[DateTime].reverse)
```